### PR TITLE
Update Text Classification with PyTorch and CMLE

### PR DESCRIPTION
**Pull request by ksalama**
This fixes issue [410](url)

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/456)

### CODE REVIEW

**Comment by thedriftofwords** [comment]
The machine type name is incorrect. (See [my comment in issue 410](https://github.com/GoogleCloudPlatform/cloudml-samples/issues/410#issuecomment-533750858)) Also, does this work in Python 3?

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb [hunk 1]
```diff
@@ -890,7 +889,6 @@
     "--origin=gs://{BUCKET}/{MODEL_DIR}/ \\\n",
     "--python-version=2.7 \\\n",
     "--runtime-version={RUNTIME_VERSION} \\\n",
-    "--framework='SCIKIT_LEARN' \\\n",
     "--package-uris=gs://{BUCKET}/{PACKAGES_DIR}/my_package-0.1.tar.gz \\\n",
     "--machine-type=mls1-c4-m4 \\\n",
```
Comment (thedriftofwords) [inline notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: The machine type name is "mls1-c4-m2"

**Comment by ksalama** [comment]
@thedriftofwords - I have applied the fixes suggested by Henry, and it seems it works fine now

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb [hunk 2]
```diff
@@ -7,8 +7,7 @@
     "id": "UWvrShlZjZwr"
    },
    "source": [
-    "# WARNING: Sample currently broken, please see [Issue #410](https://github.com/GoogleCloudPlatform/cloudml-samples/issues/410)",
-    "# Serving Text Classification Using PyTorch and AI Platform\n",
+    "# WARNING: Sample currently broken, please see [Issue #410](https://github.com/GoogleCloudPlatform/cloudml-samples/issues/410)# Serving Text Classification Using PyTorch and AI Platform\n",
```
Comment (nnegrey) [inline notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Let's remove this line now.

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb [hunk 3]
```diff
@@ -787,12 +786,11 @@
     "\n",
     "from setuptools import setup\n",
     "\n",
-    "REQUIRED_PACKAGES = ['torch', 'keras']\n",
+    "REQUIRED_PACKAGES = []\n",
```
Comment (nnegrey) [inline notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Can we remove this entirely if it's empty?

Comment (andrewferlitsch) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: @ksalama It appears that there are some changes being requested.

Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb (hunk 2 above)
Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb (hunk 1 above)
Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb (hunk 3 above)
Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

**Comment by nnegrey** [comment]
I still get an error: 
```
Creating version (this might take a few minutes)......failed.                  
ERROR: (gcloud.beta.ai-platform.versions.create) Create Version failed. Bad model detected with error:  "Failed to load model: Unexpected error when loading the model: unsupported pickle protocol: 3 (Error code: 0)"
```

**Comment by nnegrey** [comment]
Fixed versioning then got:
```
Creating version (this might take a few minutes)......failed.                  
ERROR: (gcloud.beta.ai-platform.versions.create) Create Version failed. Bad model detected with error:  "Failed to load model: Unexpected error when loading the model: problem in model_prediction - <type 'exceptions.ImportError'>: No module named _C (Error code: 0)"
```

**Comment by andrewferlitsch** [comment]
@nnegrey Any idea of the cause of error?

**Comment by ksalama** [comment]
I tested the notebook and it seems to be working fine. Couldn't reproduce the error.

**Comment by gogasca** [comment]
Closing as notebook was updated with changes

**Pull request closed without merging**